utils/setup_daos_admin.sh

```diff
@@ -55,11 +32,6 @@ echo "This script will install daos_admin for developer builds (not for producti
 echo -n "Installing $DA_SRC -> $DA_DST ... "
 chmod -x "$DA_SRC" || true
 cp "$DA_SRC" "$DA_DST"
-if [ "$SL_PREFIX" != "DAOS_LOC" ]; then
-        rpath=$(patchelf --print-rpath $DA_DST | \
-              sed "s|$SL_PREFIX|$DAOS_LOC|g")
-        patchelf --set-rpath "$rpath" $DA_DST
```

Comment:
@jolivier23 apparently this was your code but it doesn't look like it's required.  This file is mentioned in the wiki but hasn't been used in CI until now as far as I can tell.

Reply:
We shouldn't remove this.  The purpose of it is for moving from one system to another if you have to move the location of the libraries.   This fixes the rpath if daos needs to be moved because otherwise, there is no way to find libraries because LD_LIBRARY_PATH is ignored for suid executables.

Stated another way...let's say you build daos in /foo/bar/install but for whatever reason you have to install it in /bar/foo/install.   The rpaths in daos_admin need to be updated to the new location to find the requisite libraries.

You are correct that it is not used in CI